include/chimera/configuration.h

```diff
@@ -77,6 +77,16 @@ class Configuration
      */
     void AddSourcePath(const std::string &sourcePath);
 
+    /**
+     * Set whether to treat unresolvable configuration as errors.
+     */
+    void SetStrict(bool val);
+
+    /**
+     * Return whether to treat unresolvable configuration as errors.
+     */
+    bool GetStrict() const;
```

Comment:
Remove the `GetStrict()` from here and put it into the `CompiledConfiguration` instead.  This will give you a chance to resolve it in the `CompiledConfiguration` locally, use it there, then expose it as a getter `GetStrict()` for any other consumers.